src/sound.js

```diff
@@ -0,0 +1,134 @@
+import { Transport, MultibandCompressor, Loop, Player, Reverb, Filter, Volume, PanVol, start, loaded} from 'tone'
+import whistle_8_T7 from '../public/whistle/whistle_8_T7.mp3'
+import whistle_4_T3 from '../public/whistle/whistle_4_T3.mp3'
+import whistle_7_T6 from '../public/whistle/whistle_7_T6.mp3'
+import whistle_12_T11 from '../public/whistle/whistle_12_T11.mp3'
+import whistle_8_T7_B from '../public/whistle/whistle_8_T7_B.mp3'
+
+const SONGS = {
+  whistle: {
+    bpm: 70,
+    parts: [[
+      // each part consists of a set of tracks
+      // type Track: [sample, probability, introProbability?]
+      [whistle_8_T7, 1, 0],
+      [whistle_4_T3, 0.9, 1],
+      [whistle_7_T6, 0.8, 1],
+      [whistle_12_T11, 0.7, 0],
+    ], [
+      [whistle_8_T7_B, 1, 0],
+      [whistle_4_T3, 0.9, 1],
+      [whistle_7_T6, 0.8, 1],
+      [whistle_12_T11, 0.7, 0],
```

Comment:
It takes tracks in and out according to these probabilities. The melodies `whistle_8_T7` do not play at all during the intro then play 100% of the time.